include/networkit/generators/BarabasiAlbertGenerator.hpp

```diff
@@ -35,24 +35,29 @@ class BarabasiAlbertGenerator: public StaticGraphGenerator {
     Graph generateOriginal();
 
 public:
-    BarabasiAlbertGenerator();
+    BarabasiAlbertGenerator() = default;
 
     /**
      * This generator implements the preferential attachment model as introduced by Barabasi and Albert[1].
      * The original algorithm is very slow and thus, the much faster method from Batagelj and Brandes[2] is
      * implemented and the current default.
      * The original method can be chosen by setting \p batagelj to false.
-     * [1] Barabasi, Albert: Emergence of Scaling in Random Networks http://arxiv.org/pdf/cond-mat/9910332.pdf
-     * [2] ALG 5 of Batagelj, Brandes: Efficient Generation of Large Random Networks https://kops.uni-konstanz.de/bitstream/handle/123456789/5799/random.pdf?sequence=1
+     * [1] Barabasi, Albert: Emergence of Scaling in Random Networks
+     *   http://arxiv.org/pdf/cond-mat/9910332.pdf
+     * [2] ALG 5 of Batagelj, Brandes: Efficient Generation of Large Random Networks
+     *   https://kops.uni-konstanz.de/bitstream/handle/123456789/5799/random.pdf?sequence=1
      *
-     * @param k Number of attachments per node
-     * @param nMax Maximum number of nodes in the graph
-     * @param n0 Number of connected nodes to begin with
-     * @param batagelj Specifies whether to use batagelj's method or the original one; default: true
+     * The generator will emit a simple graph (given that the seed graph is simple), where all
+     * new nodes are initially connected to k random neighbors.
+     *
+     * @param k     Number of attachments per node
+     * @param nMax  Number of nodes in the graph
+     * @param n0    Number of connected nodes to begin with
+     * @param turbo Specifies whether to use Batagelj and Brandes's method rather than the naive one; default: true
      */
-    BarabasiAlbertGenerator(count k, count nMax, count n0 = 0, bool batagelj=true);
+    BarabasiAlbertGenerator(count k, count nMax, count n0 = 0, bool turbo=true);
 
-    BarabasiAlbertGenerator(count k, count nMax, const Graph& initGraph, bool batagelj=true);
+    BarabasiAlbertGenerator(count k, count nMax, const Graph& initGraph, bool turbo=true);
```

Comment:
I think it is fine to keep `batagelj`, it clarifies that we are using that algorithm. Is there a specific reason for this change?

Reply:
My intention here was to make the implications of this flag more explicit. However, I also see your point and reverted the changes (in doubt leave it as is).